Dockerfile

```diff
@@ -1,13 +1,22 @@
 FROM brew.registry.redhat.io/rh-osbs/openshift-golang-builder:rhel_9_1.21 as builder
 WORKDIR /go/src/github.com/openshift/cli-manager-operator
 COPY . .
+
+ARG OPERAND_IMAGE=quay.io/redhat-user-workloads/clio-wrklds-pipeline-tenant/clio-wrklds-pipeline/cli-manager-operator@sha256:958d000fa2a270bc09cce7150530fb78d69c74e42b0f35b22422e319fbcf871b
+ARG OPERAND_IMAGE_2=registry.redhat.io/clio-wrklds-pipeline-tenant/clio-wrklds-pipeline@sha256:958d000fa2a270bc09cce7150530fb78d69c74e42b0f35b22422e319fbcf871b
+ARG OPERAND_IMAGE_3=registry.redhat.io/clio-wrklds-pipeline/clio-wrklds-pipeline@sha256:958d000fa2a270bc09cce7150530fb78d69c74e42b0f35b22422e319fbcf871b
+ARG OPERAND_IMAGE_4=quay.io/redhat-services-prod/clio-wrklds-pipeline-cli-manager@sha256:958d000fa2a270bc09cce7150530fb78d69c74e42b0f35b22422e319fbcf871b
+ARG REPLACED_OPERAND_IMG=quay.io/openshift/origin-cli-manager:latest
+
+# Replace the operand image in deploy/07_deployment.yaml with the one specified by the OPERAND_IMAGE build argument.
+RUN find deploy/ && find deploy -type f -exec sed -i \
```

Comment:
This mimicks the pattern in https://github.com/openshift/multiarch-tuning-operator/blob/2d0956ebf69d76bc589e6c18138fcb6628e364b6/bundle.konflux.Dockerfile#L2-L12